src/darwin/Framework/CHIP/MTRBackwardsCompatShims.h

```diff
@@ -159,4 +159,15 @@ typedef NS_ENUM(uint8_t, MTROTASoftwareUpdateRequestorOTAUpdateState) {
     = 0x08,
 } MTR_DEPRECATED("Please use MTROTASoftwareUpdateRequestorUpdateState", ios(16.4, 17.2), macos(13.3, 14.2), watchos(9.4, 10.2), tvos(16.4, 17.2));
 
+/**
+ * Content Launcher cluster's StatusEnum got both the enum _and_ one of its
+ * fields renamed.  Our auto-generation does not support that, so just snapshot
+ * the pre-rename situation here.
+ */
+typedef NS_ENUM(uint8_t, MTRContentLauncherContentLaunchStatus) {
```

Comment:
Actually, maybe I can avoid this.  Commenting to block merge for an hour or so until I get a chance to try.

Reply:
I have compile errors anyway ... so this will be delayed one day for that as well.

Reply:
Fwiw, I updated the PR to remove this and just use codegen, so I don't have to manually adjust deprecation stuff later.